docker-compose.yml

```
@@ -101,6 +101,41 @@ services:
       - 5432:5432
     volumes:
       - /var/lib/postgresql/data
+  rbac:
+    image: quay.io/cloudservices/rbac:latest
+    restart: on-failure
+    environment:
+      - REDIS_PORT=6379
+      - REDIS_HOST=redis
+      - DATABASE_HOST=db
+      - DATABASE_PORT=5432
+      - DATABASE_NAME=insights
+      - DATABASE_USER=insights
+      - DATABASE_PASSWORD=insights
+      - API_PATH_PREFIX=/api/rbac
+      - DEVELOPMENT=True
+    depends_on:
+      - db
+      - redis
+    links:
+      - db
+      - redis
+    ports:
+      - 9002:8080
+    volume:
+      - ./config/devel/rbac/management/role/permissions/compliance.json:/opt/app-root/src/rbac/management/role/permissions/compliance.json:z
+      - ./config/devel/rbac/management/role/definitions/compliance.json:/opt/app-root/src/rbac/management/role/definitions/compliance.json:z
```

No point in storing the files so deeply if you can't map the folder structure in overlay:
```suggestion
      - ./config/devel/rbac/permissions.json:/opt/app-root/src/rbac/management/role/permissions/compliance.json:z
      - ./config/devel/rbac/definitions.json:/opt/app-root/src/rbac/management/role/definitions/compliance.json:z
```